fe/fe-core/src/main/java/org/apache/doris/qe/DdlExecutor.java

```diff
@@ -142,9 +142,15 @@ public static void execute(Catalog catalog, DdlStmt ddlStmt) throws Exception {
             }
         } else if (ddlStmt instanceof CancelLoadStmt) {
             boolean isAccurateMatch = ((CancelLoadStmt) ddlStmt).isAccurateMatch();
+            String label = ((CancelLoadStmt) ddlStmt).getLabel();
+            if (!isAccurateMatch && label != null) {
```

Comment:
This should be done in analyze() of CancelLoadStmt.

Reply:
done